test/JDBC/expected/BABEL_4330.out

```diff
@@ -0,0 +1,86 @@
+
+-- We should be able to use replace function in computed column.
```

Comment:
I think we need to add tests for replace as well.
Then we need to add test case to check if it is immutable
Then we should also add upgrade tests

Reply:
addressed